specification/cognitiveservices/AnomalyDetector/main.tsp

```diff
@@ -49,6 +49,8 @@ namespace AnomalyDetector;
 @doc("The secret key for your Azure Cognitive Services subscription.")
 model AnomalyDetectorApiKeyAuth is ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">;
 
+@doc("The versions of this service")
```

Comment:
For this one, just suppressing the documentation error should be sufficient

Reply:
OK, closing PR